src/components/Article.jsx

```diff
@@ -0,0 +1,50 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import Comments from "./Comments";
+
+function Article() {
+  const [article, setArticle] = useState([]);
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    axios
+      .get(`https://be-nc-news-whiy.onrender.com/api/articles/${article_id}`)
+      .then(({ data }) => {
+        setArticle([data.article]);
+      });
```

Comment:
This get request can be extracted out from this file.

While your request is fetching your data what is shown to the user? Does the user know something is happening?